0.15.1 Module build failed: Error: ENOENT

I use generator-gulp-angular. After update from 0.14.1 to 0.15.1  I get errors.

OS: Ubuntu
Typescript 1.6

```javascript
ERROR in ./src/app/index.module.ts
Module build failed: Error: ENOENT: no such file or directory, open '/home/xxx/xxx/src/app/index.config'
    at Error (native)

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:1:0 
File '/home/xxx/xxx/.tmp/typings/tsd.d.ts' not found.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:3:23 
Cannot find module './index.config'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:4:29 
Cannot find module './index.route'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:5:25 
Cannot find module './index.run'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:6:31 
Cannot find module './main/main.controller'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:7:34 
Cannot find module '../app/components/githubContributor/githubContributor.service'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:8:33 
Cannot find module '../app/components/webDevTec/webDevTec.service'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:9:27 
Cannot find module '../app/components/navbar/navbar.directive'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:10:29 
Cannot find module '../app/components/malarkey/malarkey.directive'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:13:20 
Cannot find namespace 'moment'.

ERROR in [default] /home/xxx/xxx/src/app/index.module.ts:18:2 
Cannot find name 'angular'.
```

--- a/src/deps.ts
+++ b/src/deps.ts
@@ -145,7 +145,11 @@ export class FileAnalyzer {
     resolve(resolver: IResolver, fileName: string, defPath: string): Promise<string> {
         let result;
 
-        if (!path.extname(defPath).length) {
+        if (/^[a-z0-9].*\.d\.ts$/.test(defPath)) {
+            // Make import relative
+            defPath = './' + defPath;
+            result = Promise.resolve(path.resolve(path.dirname(fileName), defPath));
+        } else {
             result = resolver(path.dirname(fileName), defPath + ".ts")
                 .error(function (error) {
                     return resolver(path.dirname(fileName), defPath + ".d.ts")
@@ -165,12 +169,6 @@ export class FileAnalyzer {
                         throw error;
                     }
                 })
-        } else {
-            if (/^[a-z0-9].*\.d\.ts$/.test(defPath)) {
-                // Make import relative
-                defPath = './' + defPath;
-            }
-            result = Promise.resolve(path.resolve(path.dirname(fileName), defPath));
         }
 
         return result

